store/tests/tests_viewset_Order.py

```diff
@@ -25,9 +25,8 @@
 from unittest import mock
 
 from blitz_api.factories import (
-    UserFactory,
-    AdminFactory,
-)
+    UserFactory, AdminFactory, RetreatFactory, RetreatTypeFactory,
+    RetreatDateFactory, OrderFactory, OptionProductFactory)
```

Comment:
For versionning reason it's often considered better to put every import on a new line, so when we edit one of them it's clear which one we edited or not